private/bufpkg/bufcheck/buflint/internal/buflintvalidate/well_known_type.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2020-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package buflintvalidate
+
+import "strings"
+
+// WellKnownTypePackage is the proto package name where all Well Known Types
+// currently reside.
+const WellKnownTypePackage string = "google.protobuf."
```

Comment:
Why is this public? None of these appear to be used outside of this package. The only thing used outside of this package is `NewValidateModule` (soon to be `NewValidateField`), nothing else should be public.

This should also be in the same block as the rest of the consts.

Reply:
100% oversight - thanks for the catch. resolved in [182d2e2](https://github.com/bufbuild/buf/pull/2377/commits/182d2e21541d6084e75fd345150fa990cde60d62)